src/cron/syncSubscriptionWithCIO.ts

```diff
@@ -28,6 +28,9 @@ const cron: Cron = {
 
         await syncSubscription([...userIds], con);
 
+        // Wait for a bit to avoid rate limiting
+        await new Promise((resolve) => setTimeout(resolve, 200));
```

Comment:
@omBratteng you can use `import { setTimeout } from 'node:timers/promises'` which returns a promise `await setTimeout(resolve, 200)`

Reply:
done.